Makefile

```diff
@@ -318,6 +320,11 @@ PORTLAYER_DEPS ?= lib/apiservers/portlayer/swagger.json \
 				  lib/apiservers/portlayer/restapi/options/*.go \
 				  lib/apiservers/portlayer/restapi/handlers/*.go
 
+$(admiralapi-client): lib/config/dynamic/admiral/swagger.json $(SWAGGER)
```

Comment:
yeah - think it would have to be vendor. Assuming the server portion of this is embedded in Admiral, the definitive version is on their side, so having it checked in independently could easily lead to disjoint definitions.